src/build/commonConfig.js

```diff
@@ -75,6 +75,12 @@ exports.extend = function (app, config) {
   config.bundleID = app.manifest.ios && app.manifest.ios.bundleID || defaultName;
   config.packageName = app.manifest.android && app.manifest.android.packageName || defaultName;
 
+  var ios = app.manifest.ios;
+  if (ios) {
+    config.widgetID = ios.widgetID || "widget";
+    config.widgetGroup = ios.widgetGroup || "group." + config.bundleID;
+  }
```

Comment:
Based on the discussion, we might be able to make this happen with changes in build.js of native-ios only.

Waiting for you to confirm